src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
+  const newSeparator = toFormat.slice(3);
+  const splitDate = date.split(oldSeparator);
```

Comment:
The `split` method expects a string as the separator, but `oldSeparator` is an array because of the incorrect use of `slice` on line 11. After fixing the `slice` call, make sure to pass the actual string separator to the `split` method.